README.md

```diff
@@ -1,2 +0,0 @@
-# dynamodb-sink
```

Comment:
As discussed, if README.rst is available, it would be shown on the home page of the repo. Hence, not creating a duplicate README.md file.